docs/src/main/sphinx/admin/logging.md

```diff
@@ -0,0 +1,131 @@
+
+# Logging
+
+Trino include numerous features to better understand and monitor a running
+system, such as [](/admin/opentelemetry) or [](/admin/jmx). Logging and
+configuring logging is one important aspect for operating and troubleshooting
+Trino.
+
+## Configuration
+
+Trino application logging is optional and configured in the `log.properties`
+file in your Trino installation `etc` configuration directory as set by the
+[launcher](running-trino).
+
+Use it to add specific loggers and configure the minimum log levels. Every
+logger has a name, which is typically the fully qualified name of the class that
+uses the logger. Loggers have a hierarchy based on the dots in the name, like
+Java packages. The four log levels are `DEBUG`, `INFO`, `WARN` and `ERROR`,
+sorted by decreasing verbosity.
+
+For example, consider the following log levels file:
+
+```properties
+io.trino=WARN
+io.trino.plugin.iceberg=DEBUG
+io.trino.parquet=DEBUG
+```
+
+The preceding configuration sets the changes the level for all loggers in the
+`io.trino` namespace to `WARN` as an update from the default `INFO` to make
+logging less verbose. The example also increases logging verbosity for the
+Iceberg connector using the `io.trino.plugin.iceberg` namespace, and the Parquet
+file reader and writer support located in the `io.trino.parquet` namespace to
+`DEBUG` for troubleshooting purposes. 
+
+Additional loggers can include other package namespaces from libraries and
+dependencies embedded within Trino or part of the Java runtime, for example:
+
+* `io.airlift` for the [Airlift](https://github.com/airlift/airlift) application
+  framework used by Trino.
+* `org.eclipse.jetty` for the [Eclipse Jetty](https://jetty.org/) web server
+  used by Trino.
+* `org.postgresql` for the [PostgresSQL JDBC driver](https://github.com/pgjdbc)
+  used by the PostgreSQL connector.
+* `javax.net.ssl` for TLS from the Java runtime.
+* `java.io` for I/O operations.
+
+There are numerous additional properties available to customize logging in
+[](config-properties), with details documented in [](/admin/properties-logging)
+and in following example sections.
+
+## Log output
+
+By default, logging output is file-based with rotated files in `var/log`:
+
+* `launcher.log` for logging out put from the application startup from the
```

Comment:
It might be worth noting this file is only used if launcher starts Trino in the background, which is _not_ the case when running Trino in a container, where it runs in the foreground.

Reply:
Added .. thanks for that info